rust/src/event.rs

```diff
@@ -239,6 +249,17 @@ mod test_event {
             },
             None => assert!(false),
         }
+
+        let web_vitals = get_web_vital();
+        let mut web_vitals_buffer: Vec<u8> = Vec::new();
+        web_vitals.encode(&mut web_vitals_buffer).unwrap();
+        match Event::decode(&EventModel::WebVitals, &*web_vitals_buffer) {
```

Comment:
```suggestion
        match Event::decode(&EventModel::WebVitals, web_vitals_buffer) {
```
With `*`, you dereference `web_vitals_buffer`, and you re-reference it with `&`.
Basically, `&*` cancel each other, so you can remove them.

In several other repos, the CI started to yell at us because of this pattern as soon as we switched to rust 1.65+, so I'm surprised it doesn't complain in this case.